packages/grid/_modules_/grid/constants/i18nConstants.ts

```diff
@@ -0,0 +1,7 @@
+export const DEFAULT_LOCALE_TEXT = {
+  densityText: 'Density',
+  densityLabelText: 'Density',
+  densityOptionTextCompact: 'Compact',
+  densityOptionTextStandard: 'Standard',
+  densityOptionTextComfortable: 'Comfortable',
```

Comment:
Do we need to mention "text" in the key? I think that the name of the prop is enough. The benchmark links can give some inspiration on the naming convention. For instance, some use a nested structure, ag-grid is flat on the other hand.

Maybe we could also prefix with the context "toolbar" and be shorter on the place where it's used, focused more on the actual meaning:

```suggestion
  toolbarDensity: 'Density',
  toolbarDensityLabel: 'Density',
  toolbarDensityCompact: 'Compact',
  toolbarDensityStandard: 'Standard',
  toolbarDensityComfortable: 'Comfortable',
```

Reply:
I think I like this more, It does bring more context regarding where that locale text is being used